Adyen/Utilities/Environment.swift

```diff
@@ -18,12 +18,16 @@ public struct Environment: APIEnvironment {
     /// :nodoc:
     public var queryParameters: [URLQueryItem] {
         guard let clientKey = clientKey else { return [] }
-        return [URLQueryItem(name: "token", value: clientKey)]
+        return [URLQueryItem(name: "clientKey", value: clientKey)]
     }
     
     /// :nodoc:
     public var clientKey: String?
-    
+
+    /// Adyen's local environment.
+    /// :nodoc:
+    public static let local = Environment(baseURL: URL(string: "http://localhost:8080/")!)
```

Comment:
Not sure if the local dev environment in public SDK code is a good idea..
It is good for us, but It is not likely our merchants can actually enjoy the thing :D

What en odds that someone will have local API perfectly mimicking our APIs on exactly 8080 port ?

Reply:
its good for us only, that's why I added `/// :nodoc:`, it has no effect good or bad for the merchants IMO